python/glow/gwas/tests/test_log_reg.py

```diff
@@ -13,7 +13,14 @@ def run_score_test(genotype_df, phenotype_df, covariate_df, fit_intercept=True):
     Y = phenotype_df.to_numpy(copy=True)
     Y_mask = ~np.isnan(Y)
     Y[~Y_mask] = 0
-    state = lr._create_log_reg_state(Y, pd.DataFrame(), None, C, Y_mask)
+    state_rows = [
+        lr._prepare_one_phenotype(C, pd.Series({
+            'label': c,
+            'values': phenotype_df[c]
+        })) for c in phenotype_df
+    ]
+    phenotype_names = phenotype_df.columns.to_series().astype('str')
+    state = lr._pdf_to_log_reg_state(pd.DataFrame(state_rows), phenotype_names, C.shape[1])
```

Comment:
Why didn't we use `_create_log_reg_state ` here?

Reply:
I wanted to be able to run a basic test without using Spark. It's much easier to experiment and debug when Spark isn't involved.